src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -145,10 +145,21 @@ int halide_hexagon_remote_set_performance(
     unsigned int bwMegabytesPerSec,
     unsigned int busbwUsagePercentage,
     int set_latency,
-    int latency) {
+    int latency,
+    int powerlevel) {
 
     HAP_power_request_t request;
-
+    const uint32_t numPowerLevels = 7;
+    const HAP_dcvs_voltage_corner_t voltageCorner[numPowerLevels]
```

Comment:
Seems safer/easier to remote numPowerLevels above, change this to:

    const HAP_dcvs_voltage_corner_t voltage_corners[] = {
        ...
        num_power_levels,
    };